src/multivariates.jl

```diff
@@ -113,21 +113,11 @@ end
 
 ## log likelihood
 
-function _loglikelihood(d::MultivariateDistribution, X::AbstractMatrix)
-    ll = 0.0
-    for i in 1:size(X, 2)
-        ll += _logpdf(d, view(X,:,i))
-    end
-    return ll
-end
-
 function loglikelihood(d::MultivariateDistribution, X::AbstractMatrix)
-    size(X,1) == length(d) ||
-        throw(DimensionMismatch("Inconsistent array dimensions."))
-    _loglikelihood(d, X)
+    size(X, 1) == length(d) || throw(DimensionMismatch("Inconsistent array dimensions."))
+    return sum(x -> _logpdf(d, x), X, 2)
```

Comment:
The problem with this is that `x` is a scalar, but `_logpdf` wants `x` to be an array. I guess that's why it was using a custom sum here, so it could take slices rather than individual elements. Ref https://github.com/JuliaLang/METADATA.jl/pull/8848#issuecomment-294328801